pkg/agent/controller/traceflow/traceflow_controller.go

```diff
@@ -536,6 +536,9 @@ func (c *Controller) preparePacket(tf *crdv1alpha1.Traceflow, intf *interfacesto
 		if tf.Spec.Packet.TransportHeader.TCP.Flags != 0 {
 			packet.TCPFlags = uint8(tf.Spec.Packet.TransportHeader.TCP.Flags)
 		}
+		if !liveTraffic && tf.Spec.Packet.TransportHeader.TCP.Flags == 0 {
```

Comment:
this code needs to be revisited for this PR:
https://github.com/antrea-io/antrea/blob/01817a5d221d86d2c26c629270b0d09c0b98294c/pkg/agent/controller/traceflow/traceflow_controller.go#L500-L504

The comment is no longer relevant and it is confusing to set the TCP flags in 2 places.

Reply:
Remove this part since we always set it as 2 by default.

Reply:
So I guess by setting the TCP flags here, it is not possible for the users to force the flags to be 0. I don't know if this is an issue at all though, since Traceflow is unlikely to be useful when the SYN flag is not set.

@tnqn what do you think? When we promote the API to beta, should be make the `Flags` field a pointer instead (`*int32`)?

Reply:
tcp_flags 0 seems only useful when we want to verify the pipeline can drop TCP NULL scan. I'm fine with changing the field to pointer to make the use case supported.